ref/appliance-command-reference/monitor.md

```diff
@@ -0,0 +1,40 @@
+The `monitor` command allows managing the monitoring dashboard. You can expose or unexpose the monitoring dashboard using the subcommands. By default, the monitoring dashboard is exposed.
+
+    anbox-cloud-appliance monitor <subcommand>
+
+## Subcommands
+
+### `account`
+Manages accounts that can access the monitoring dashboard.
+
+    anbox-cloud-appliance monitor account <subcommand>
+
+#### Subcommands
+
+##### `create`
+Creates a new account to access the monitoring dashboard. The monitoring dashboard is configured with basic authentication and hence you need to create an account to access it.
+
+    anbox-cloud-appliance monitor account create <account-name> [--password=<value>] [--email=<email>] [--role=viewer|admin|editor] [options]
+
+###### Options
+
+|Option | Description |
+|-------|-------------|
+| `-e`, `--email <email>`|Email address of the monitor account|
+| `-p`, `--password <value>`|Password of the monitor account|
+| `-r`, `--role <value>`|Role of the monitor account can be either of these three options: viewer/admin/editor. The default role is viewer.|
+
+##### `delete`
+Deletes the specified account and removes access to the monitoring dashboard for the mentioned account.
+
+    anbox-cloud-appliance monitor account delete <account-name>
+
```

Comment:
Same question as above: what do you think about combining these subcommands into two commands: `account create` and `account delete`?